packages/babel/README.md

```diff
@@ -0,0 +1,368 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-babel
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-babel
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-babel
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-babel
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-babel
+
+🍣 Seamless integration between Rollup and Babel.
+
+## Why?
```

Comment:
@shellscape Should this section be moved down?

Reply:
I was thinking where this should be put when I was tweaking the README and I've decided to keep it here, at the top, as it made the most sense for me. Can, of course, change this but this acts kinda as an introduction to the whole thing, so it made sense to keep it at the beginning

Reply:
I would say in 99% of cases, this is "meta" and should go after the really important bits (reqs, install, use, options). but the babel plugin does require some "do you really need this" knowledge up front. I'm good leaving this here as an exception.